impeller/compiler/BUILD.gn

```diff
@@ -68,6 +70,7 @@ impeller_component("compiler_lib") {
     "//flutter/fml",
 
     # All third_party deps must be included by the global license script.
+    "//flutter/impeller/shader_bundle:shader_bundle_flatbuffers",
```

Comment:
The comment on the line above seems to be referring to the `third_party` deps on the line below. Maybe this line should instead be grouped right below `//flutter/fml` instead of below this comment?

Reply:
Fixed